website/docs/patterns/form-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,229 @@
+| [Textarea](/components/form/textrea) | Lorem ipsum | Messages, longer-form content |
+
+#### Data fields
+
+<!-- TODO -->
+
+| Form control | Usage | Examples |
+|--------------|-------|----------|
+| [Checkbox](/components/form/checkbox) | Lorem ipsum | Lorem ipsum |
+| [Radio](/components/form/radio) | Lorem ipsum | Lorem ipsum |
+| [Radio Card](/components/form/radio-card) | Lorem ipsum | Lorem ipsum |
+| [Select](/components/form/select) | Lorem ipsum | Lorem ipsum |
+| [Toggle](/components/form/toggle) | Lorem ipsum | Lorem ipsum |
+
+### Groups
+
+Fields may be **grouped** together either horizontally or vertically if they are collecting related information or information that is part of the same object. Within a group there should bea  16px gap either horizontally (creating multiple columns), or vertically between each field.
+
+![Example of fields grouped together](/assets/patterns/form-patterns/field-groups.png =450x*)
+
+!!! Dont
+
+Don't group (either horizontally or vertically) field types that are related but not of the same type, e.g., a text input and a toggle.
+
+![Grouping different field types](/assets/patterns/form-patterns/different-field-type.png)
+!!!
+
+Common grouping of fields can include:
+
+- Credit card information; card number, expiration data, security code.
+- Multi-line address field
+- First and last name
+
+#### Alignment
+
+Fields organized in a horizontal group should be aligned to the baseline of each element to account for fields which have helper text.
+
+![Baseline alignment within a group](/assets/patterns/form-patterns/baseline-alignment.png =500x*)
+
+#### Multi-column layout
+
+Grouping form elements horizontally creates a multi-column layout within the form. When grouping elements,
+
+- use a consistent number of columns throughout the form; e.g., don't organize a group of fields in two-columns and another group in three-columns.
+- don't exceed more than three fields in a horizontal group.
+- fields should stack vertically when the width of the viewport and form container shrinks.
+
+Multi-column layouts can be impacted by the width of the overall form. Use grouping logically based on the layout, width of the page, and overall UX strategy in the application.
+
+### Text
+
+Sections consisting of text should use a 8px vertical gap between text elements.
+
+![Example of a text section](/assets/patterns/form-patterns/text-section.png =600x*)
+
+Text elements within a form should use logical, step-based sizing to reinforce hierarchy within sections, as well as within the form itself. While specifics around type hierarchy should be determined at the application level, adhering to these guidelines will help establish consistency both at the page level, and when constrained within another element or component.
+
+<!-- TODO -->
+
+### Actions
+
+#### Button grouping
+
+Buttons should be grouped using the [ButtonSet](/components/button-set) guidelines, e.g., using a 16px horizontal gap between each button.
```

Comment:
```suggestion
Buttons should be grouped using the [ButtonSet](/components/button-set) guidelines, e.g. using a 16px horizontal gap between each button.
```

Reply:
There are also other similar occurrences in the text, so is clearly intentional. Maybe double check with @KristinLBradley (I am not sure I've ever seen it used this way)

Reply:
I believe this was how @heatherlarsen was using "e.g." when part of a sentence. Is this right? ([For reference](https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/])

Reply:
I'm no grammar expert, but this is what Grammarly corrects it to, so that's what I've been using across the docs. The link @jorytindall shared has more details.

Reply:
I would go by Grammarly and the linked reference on this one

Reply:
Yes, the comma should stay there. 👍

Reply:
OK, it's an American English vs British English thing. All good then :)

<img width="677" alt="screenshot_2442" src="https://user-images.githubusercontent.com/686239/223101580-4f9482b7-5950-4219-a993-c4ada13e8fd4.png">